src/components/VideoCard.js

```diff
@@ -4,11 +4,12 @@ const VideoCard = ({ info }) => {
   
 
   const { snippet, statistics } = info;
+  console.log(snippet)
   return (
-    <div className="w-96 m-3 p-3">
+    <div className="m-2 p-2">
       <img
         alt="thumbnail"
-        className="rounded-lg"
+        className="rounded-lg w-[360px]"
```

Comment:
define 360px in tailwind config

Reply:
I have not defined it separately in the config file as it is used only once.